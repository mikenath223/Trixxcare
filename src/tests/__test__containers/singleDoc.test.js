import React from 'react';
import withReduxRouter from '../../utils/testUtils';
import SingleDoc from '../../containers/DocProfile/SingleDoc';


describe('SingleDoc', function () {
  beforeEach(() => {
    this.props = {
      match: {
        params: {
          doc: 'Matt%20P.%20Abel,%20M.D.-2',
        },
      },
    };
  });

  it('should match snapshot', () => {
    const { container } = withReduxRouter(
      <SingleDoc {...this.props} />,
    );

    expect(container).toMatchSnapshot();
  });
});
