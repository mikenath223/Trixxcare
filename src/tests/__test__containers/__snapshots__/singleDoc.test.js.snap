// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SingleDoc should match snapshot 1`] = `
<div>
  <div
    class="containerLoad container"
  >
    <img
      alt=""
      src="https://www.ecoloxtech.com/images/giphycolor.gif"
    />
  </div>
</div>
`;
